util/appveyor/build.cmd

```diff
@@ -0,0 +1,21 @@
+@echo off
+:: To build extensions for 64 bit Python 3, we need to configure environment
+:: variables to use the MSVC 2010 C++ compilers from GRMSDKX_EN_DVD.iso of:
+:: MS Windows SDK for Windows 7 and .NET Framework 4
+::
+:: More details at:
+:: https://github.com/cython/cython/wiki/64BitCythonExtensionsOnWindows
```

Comment:
Found my source! Really should have linked this instead, either I copy/pasted the wrong thing or that used to be in the article at the link too - https://packaging.python.org/appveyor/

At the bottom. Will update the link if you're happy with the current solution?